From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@debian.org>
Date: Sun, 18 Sep 2016 11:26:45 +0200
Subject: Generate docs with modern linuxdoc

---
 doc/LinuxDocTools.pm |  2 +-
 doc/Makefile         | 14 +++++++++-----
 2 files changed, 10 insertions(+), 6 deletions(-)

diff --git a/doc/LinuxDocTools.pm b/doc/LinuxDocTools.pm
index 51d4a04..b6c950a 100644
--- a/doc/LinuxDocTools.pm
+++ b/doc/LinuxDocTools.pm
@@ -372,7 +372,7 @@ sub process_file
         }
     }
   #
-  my($precmd) = "|sgmlpre output=$global->{format} $global->{define}";
+  my($precmd) = "|/usr/lib/linuxdoc-tools/sgmlpre output=$global->{format} $global->{define}";
 
   #
   #  You can hack $NsgmlsOpts here, etcetera.
diff --git a/doc/Makefile b/doc/Makefile
index f88c320..1e5f4ba 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -12,8 +12,8 @@ endif
 .PHONY: prog.sgml bird.sgml
 
 docs: progdocs userdocs
-progdocs: prog.html prog.ps
-userdocs: bird.html bird.ps
+progdocs: prog.html prog.pdf
+userdocs: bird.html bird.pdf
 
 prog.sgml:
 	$(srcdir)/tools/progdoc $(srcdir_abs)
@@ -22,8 +22,12 @@ prog.sgml:
 	./sgml2html $<
 
 %.dvi: %.tex
-	latex $<
-	latex $<
+	TEXINPUTS=.:tex: latex $<
+	TEXINPUTS=.:tex: latex $<
+
+%.pdf: %.tex
+	TEXINPUTS=.:tex: pdflatex $<
+	TEXINPUTS=.:tex: pdflatex $<
 
 %.ps: %.dvi
 	dvips -D600 -ta4 -o $@ $<
@@ -43,4 +47,4 @@ clean:
 	rm -f prog.sgml
 
 distclean: clean
-	rm -f *.html *.ps
+	rm -f *.html *.ps *.pdf
