From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@debian.org>
Date: Sun, 18 Sep 2016 11:26:45 +0200
Subject: Generate docs with modern linuxdoc

---
 doc/LinuxDocTools.pm |  2 +-
 doc/Makefile         | 14 +++++++++-----
 2 files changed, 10 insertions(+), 6 deletions(-)

--- bird.orig/doc/LinuxDocTools.pm
+++ bird/doc/LinuxDocTools.pm
@@ -372,7 +372,7 @@ sub process_file
         }
     }
   #
-  my($precmd) = "|sgmlpre output=$global->{format} $global->{define}";
+  my($precmd) = "|/usr/lib/linuxdoc-tools/sgmlpre output=$global->{format} $global->{define}";
 
   #
   #  You can hack $NsgmlsOpts here, etcetera.
--- bird.orig/doc/Makefile
+++ bird/doc/Makefile
@@ -12,7 +12,7 @@ endif
 .PHONY: prog.sgml bird.sgml
 
 docs: progdocs userdocs
-progdocs: prog.html prog.ps
+progdocs: prog.html prog.pdf
 userdocs: bird.html bird.pdf
 
 prog.sgml:
@@ -22,8 +22,12 @@ prog.sgml:
 	./sgml2html $<
 
 %.dvi: %.tex
-	latex $<
-	latex $<
+	TEXINPUTS=.:tex: latex $<
+	TEXINPUTS=.:tex: latex $<
+
+%.pdf: %.tex
+	TEXINPUTS=.:tex: pdflatex $<
+	TEXINPUTS=.:tex: pdflatex $<
 
 %.ps: %.dvi
 	dvips -D600 -ta4 -o $@ $<
@@ -47,4 +51,4 @@ clean:
 	rm -f prog.sgml
 
 distclean: clean
-	rm -f *.html *.ps
+	rm -f *.html *.ps *.pdf
