From: =?utf-8?b?T25kxZllaiBTdXLDvQ==?= <ondrej@debian.org>
Date: Sun, 18 Sep 2016 11:26:45 +0200
Subject: Link using ld

---
 tools/Rules.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/Rules.in b/tools/Rules.in
index d177ed1..bf5a404 100644
--- a/tools/Rules.in
+++ b/tools/Rules.in
@@ -69,8 +69,8 @@ subdir: all.o
 all.o: $(objs)
 #       $(LD) -r -o $@ $^
 #	Changed to $(CC) because $(LD) has problems with crosscompiling
-	@echo LD -r -o $@ $^
-	@$(CC) -nostdlib -r -o $@ $^
+	@echo LD $(CFLAGS) -r -o $@ $^
+	@$(CC) $(CFLAGS) -nostdlib -r -o $@ $^
 
 endif
 
