diff --git a/configure b/configure
index 0e784ef..665afe1 100755
--- a/configure
+++ b/configure
@@ -638,6 +638,7 @@ ac_includes_default="\
 
 ac_subst_vars='LTLIBOBJS
 LIBOBJS
+SUFFIX6
 CLIENT_LIBS
 CLIENT
 EGREP
@@ -1934,11 +1935,13 @@ esac
 
 if test "$enable_ipv6" = yes ; then
 	ip=ipv6
+	SUFFIX6=6
 	if test "$with_protocols" = all ; then
 		with_protocols=bgp,pipe,rip,static
 	fi
 else
 	ip=ipv4
+	SUFFIX6=""
 	if test "$with_protocols" = all ; then
 		with_protocols=bgp,ospf,pipe,rip,static
 	fi
@@ -4333,7 +4336,7 @@ if test "${bird_cv_sys_linux_version+set}" = set; then
 else
 
 cat > conftest.$ac_ext <<EOF
-#line 4336 "configure"
+#line 4339 "configure"
 #include "confdefs.h"
 #include <linux/version.h>
 VERSION: UTS_RELEASE
@@ -4363,7 +4366,7 @@ if test "${bird_cv_sys_linux_version+set}" = set; then
 else
 
 cat > conftest.$ac_ext <<EOF
-#line 4366 "configure"
+#line 4369 "configure"
 #include "confdefs.h"
 #include <linux/version.h>
 VERSION: UTS_RELEASE
